docs/howto/code/flask-async/task.yaml

```diff
@@ -89,8 +93,6 @@ execute: |
   charmcraft init --profile flask-framework --name flask-async-app
   # [docs:charm-init-end]
 
-  sed -i "s/paas-charm.*/https:\/\/github.com\/canonical\/paas-charm\/archive\/async-workers.tar.gz/g" requirements.txt
```

Comment:
how come?

Reply:
The file no longer exists so that line was causing the spread test to fail.